gwtquery-core/src/main/java/com/google/gwt/query/client/GQuery.java

```diff
@@ -133,12 +130,12 @@ public TagWrapper(int wrapDepth, String preWrap, String postWrap) {
   /**
    * A Browser object with a set of browser-specific flags.
    */
-  public static final Browser browser = GWT.isClient() ? GWT.<Browser>create(Browser.class) : null;
+  public static final Browser browser = GWT.isClient() ? GWT.<Browser> create(Browser.class) : null;
```

Comment:
That was introduced by formatter, rolled back these lines. Checkstyle does not complains.